### PR TITLE
added new method

### PR DESCRIPTION
**Pull request by brvtl**
again!

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -181,6 +181,11 @@ class Project < ActiveRecord::Base
     clone
   end
 
+  def test
+    puts "blarpants"
```
Comment (glennr) [inline app/models/project.rb]: change to "blar" please

**Comment by glennr** [comment]
+1

**Comment by brvtl** [comment]
:shipit:

**Comment by glennr** [comment]
wait! merge dev back in first!

**Comment by glennr** [comment]
by dev i mean this test branch gr/17628085_fix_classification_errors

**Pull request merged**